wrappers/image/Image.jsx

```diff
@@ -12,8 +12,9 @@ const ImageWrapped = ({
   priority,
   className,
   quality = 75,
+  isCropped = false,
 }) => {
-  const src = getImageUrl(photo, `${width}x${height}`, type);
+  const src = getImageUrl(photo, `${width}x${height}${isCropped ? '__cropped' : ''}`, type);
```

Comment:
nie, spravme to tak, ze parameter nazves `suffix` a pridas ho tam nasledne. Tym padom tam budeme vediet posielat `__cropped`, alebo `@2x`, `@3x` alebo cokolvek nas napadne v buducnosti

Reply:
@ivankopcik zapracované